src/kvstore/RocksEngine.cpp

```diff
@@ -182,6 +276,35 @@ ResultCode RocksEngine::removeRange(const std::string& start,
 }
 
 
+ResultCode RocksEngine::removePrefix(const std::string& prefix) {
+    rocksdb::Slice pre(prefix.data(), prefix.size());
```

Comment:
We should use deleteRange.

Reply:
I'm not convinced yet. Especially in the rocksdb version we are using, DeleteRange is an experimental feature, which we should not use

After we upgrade the rocksdb and benchmark to prove DeleteRange is faster, we can certainly switch to it

Reply:
Fine, open another issue to track it.